syntax = "proto3";

package proto.health.v1;

service HealthService {
  rpc HealthCheck(HealthCheckRequest) returns (HealthCheckResponse) {}
}

message HealthCheckRequest {}

message HealthCheckResponse {
  string status = 1;
  string version = 2;
}
